homework-solutions/js/hw-4/task-2.js

```diff
@@ -14,3 +14,23 @@
   5. Проверить задание со следующими значениями в переменной age: 10, 17, 18, 19, 59, 60, 61
 
   */
+
+  let minAge;
+  let maxAge;
+  let age;
+  
+  minAge = 18;
+  maxAge = 60;
+  age = '17';
+  
+  if (typeof age != 'number') console.log('Incorrect data type');
+  
+  if (age < minAge) {
+    console.log("You don't have access cause your age is " + age + " It's less then " + minAge);
```

Comment:
В таком решении скрипт продолжит вьіполнение

```suggestion
  if (typeof age != 'number')  {
     console.log('Incorrect data type');
  } else {
  if (age < minAge) {
    console.log("You don't have access cause your age is " + age + " It's less then " + minAge);
```